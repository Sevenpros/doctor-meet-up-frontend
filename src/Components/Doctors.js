import React from 'react';

const Doctors = () => (
  <h1>DOCTORS LIST</h1>
);

export default Doctors;
