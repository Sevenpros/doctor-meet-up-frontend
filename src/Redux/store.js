import { configureStore } from '@reduxjs/toolkit';
// import logger from 'redux-logger';
import doctorsReducer from './doctors/doctors';
import doctorReducer from './doctors/doctor';

const store = configureStore({
  reducer: {
    doctors: doctorsReducer,
    doctor: doctorReducer,
  },
  // middleware: (getDefaultMiddleware) => getDefaultMiddleware().concat(logger),
});
export default store;
