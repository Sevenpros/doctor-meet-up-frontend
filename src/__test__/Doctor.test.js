import { Provider } from 'react-redux';
import renderer from 'react-test-renderer';
import { BrowserRouter as Router } from 'react-router-dom';
import store from '../redux/store';

/* eslint-disable-next-line ... */
import Doctors from '../Components/Doctors';

describe('Bookings tests', () => {
  it('render Countries test', () => {
    const tree = renderer.create(
      <Provider store={store}>
        <Router>
          <Doctors />
        </Router>
      </Provider>,

    );
    expect(tree).toMatchSnapshot();
  });
});
