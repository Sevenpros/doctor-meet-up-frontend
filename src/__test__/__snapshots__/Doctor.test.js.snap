// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Bookings tests render Countries test 1`] = `
<div
  className="main-container container"
>
  <div
    className="header"
  >
    <h1>
      Available Doctors
    </h1>
    <p>
      Please select a Doctor
    </p>
  </div>
  <div
    className="sc-jSUZER dQEjUh rec rec-carousel-wrapper doctors"
    style={Object {}}
  >
    <div
      className="sc-eDvSVe lguFiu rec rec-carousel"
      size={
        Object {
          "height": 0,
        }
      }
      style={
        Object {
          "height": 0,
        }
      }
    >
      <button
        className="sc-bcXHqe Slxdj rec rec-arrow rec rec-arrow-left"
        disabled={true}
        onClick={[Function]}
        style={Object {}}
        tabIndex={0}
        type="button"
      >
        ❮
      </button>
      <div
        className="sc-dkrFOg biYdFZ rec rec-slider-container"
      >
        <div
          className="sc-hLBbgP ikrMvi rec rec-slider"
          style={
            Object {
              "left": "0px",
              "right": "auto",
              "top": "auto",
              "transition": "all 500ms ease",
            }
          }
        >
          <div
            className="rec rec-swipable"
            onMouseDown={[Function]}
            style={
              Object {
                "display": "flex",
                "flexDirection": "row",
              }
            }
          />
        </div>
      </div>
      <button
        className="sc-bcXHqe Slxdj rec rec-arrow rec rec-arrow-right"
        disabled={true}
        onClick={[Function]}
        style={Object {}}
        tabIndex={0}
        type="button"
      >
        ❯
      </button>
    </div>
    <div
      className="sc-iBYQkv jtKTCe rec rec-pagination"
    >
      <button
        className="sc-gKPRtg cVXxbE rec rec-dot rec rec-dot_active"
        onClick={[Function]}
        tabIndex={-1}
        type="button"
      />
    </div>
  </div>
</div>
`;
